card.py

```diff
@@ -1,8 +1,26 @@
 class Card:
+    face_value_mapping = {
+        1: 'A',
+        11: 'J',
+        12: 'Q',
+        13: 'K',
+    }
+
     def __init__(self, v, s):
-        self.value = v
         self.suit = s
+        self.value = v
 
-    def __repr__(self):
-        return f'{self.value} of {self.suit}'
+    @property
+    def score(self):
+        if self.value > 10:
+            return 10
+        elif self.value == 1:
+            return 11
+        return self.value
+
+    @property
```

Comment:
This is exactly what I was thinking. It's a little silly for something like this, but for `score` it makes way more sense. It's a way of accessing calculated data on an instance of a class while treating it like a field instead of a method. AKA, it makes using this data easier else where.